backend/plugins/frontline_api/src/modules/integrations/call/db/models/Customers.ts

```
@@ -0,0 +1,126 @@
+  class Customer {
+    public static async checkDuplication(
+      primaryPhone: string,
+      idsToExclude?: string[] | string,
+    ) {
+      const query = {} as any;
+      let previousEntry;
+
+      // Adding exclude operator to the query
+      if (idsToExclude) {
+        query._id =
+          idsToExclude instanceof Array
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)